lib/services/vote_service.dart

```diff
@@ -13,30 +15,49 @@ class VoteService {
   VoteService._();
 
   final ItemApi _itemApi = new ItemApi();
+  final CommentApi _commentApi = new CommentApi();
   CachedVoteRepository _voteRepository = CachedVoteRepository.instance;
 
-  Future voteItem(Item item, Vote vote) async {
+  Future<String> get nonce async {
```

Comment:
Make this a function. It's very odd to await a getter.

Reply:
fixed with 356cc30